components/admin-panel/sections/ActivityLog/index.js

```diff
@@ -40,21 +38,47 @@ const activityLogQuery = gqlV2/* GraphQL */ `
     $dateFrom: DateTime
     $dateTo: DateTime
     $type: [ActivityAndClassesType!]
-    $attribution: ActivityAttribution
+    $filteredAccount: [AccountReferenceInput!]
   ) {
     account(slug: $accountSlug) {
       id
       isHost
       type
+      ... on Collective {
+        childrenAccounts {
+          nodes {
+            id
+            legacyId
+            name
+            slug
+            imageUrl
+            legacyId
+          }
+        }
+      }
+      ... on Organization {
+        memberOf(role: HOST) {
+          nodes {
+            id
+            account {
+              id
+              legacyId
+              name
+              slug
+              imageUrl
+              legacyId
+            }
+          }
+        }
+      }
```

Comment:
:no_entry_sign: This is paginated, so this approach will miss some accounts. And even if we set a high limit, fetching 3000+ collectives for OSC will have a huge cost. Can we rather rely on the async capabilities of the collective picker and use its `hostCollectiveIds` prop?

Reply:
I've used the `hostCollectiveIds` prop to eliminate the fetching of hosted accounts. But we still need the `childrenAccounts` and hence I've kept that part. Do you see any other method of eliminating fetching the `childrenAccounts` as well. On the API side we fetch 100 child accounts by default. I thought this was a large enough limit, or is there a more optimal method of fetching the child collectives as well? 🤔

Reply:
Maybe introduce a `parentCollectiveId` parameter on `search`? If it's a host, we'll search using `hostCollectiveIds`m otherwise using `parentCollectiveId`

Reply:
Thanks. I've done this and seems to work correctly.